.env.local.bak

```
@@ -0,0 +1,106 @@
+KAFKA_CLIENT_ID=chemchat-dev
+KAFKA_GROUP_ID=chemchat-consumers-dev
+
+# Elasticsearch Configuration
+ELASTICSEARCH_URL=http://localhost:9200
+ELASTICSEARCH_INDEX_PREFIX=chemchat_dev
+
+# JWT Secrets (Development - Replace in production)
+JWT_ACCESS_SECRET=dev-super-secret-access-token-key-change-in-production
+JWT_REFRESH_SECRET=dev-super-secret-refresh-token-key-change-in-production
+JWT_WS_SECRET=dev-super-secret-websocket-token-key-change-in-production
+JWT_MFA_SECRET=dev-super-secret-mfa-token-key-change-in-production
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_MFA_SECRET key should go before the JWT_REFRESH_SECRET key

(UnorderedKey)